op-batcher/metrics/noop.go

```diff
@@ -46,3 +47,4 @@ func (*noopMetrics) RecordBlobUsedBytes(int) {}
 func (*noopMetrics) StartBalanceMetrics(log.Logger, *ethclient.Client, common.Address) io.Closer {
 	return nil
 }
+func (nm *noopMetrics) PendingDABytes() float64 { return math.MaxFloat64 } // force permanent throttling state for testing
```

Comment:
```suggestion
func (*noopMetrics) PendingDABytes() float64 { return math.MaxFloat64 } // force permanent throttling state for testing
```
but not sure it's a good idea to add this to the `noopMetrics`. If we need this specific behavior in a test, we should rather create a mock there, possibly by just embedding the noopMetrics and overwriting this function.

Reply:
thanks, yes, this is for my in-progress e2e test, will convert it to a mock.

Reply:
done